templates/base.html

```diff
@@ -41,7 +41,7 @@
 
       <footer>
         {% block footer %}
-          {{ macros::mini_logo(classes="", title=config.title, siteurl=config.base_url, logourl=config.extra.profile) }}
+          {{ macros::mini_logo(classes="", title=config.title, siteurl=base_url, logourl=config.extra.profile) }}
```

Comment:
Whoops, `base_url` isn't defined here. This causes issues when extending this file.